superset-frontend/src/dashboard/components/Dashboard.jsx

```diff
@@ -59,13 +59,11 @@ const propTypes = {
   ownDataCharts: PropTypes.object.isRequired,
   layout: PropTypes.object.isRequired,
   impressionId: PropTypes.string.isRequired,
-  initMessages: PropTypes.array,
```

Comment:
I was not able to find any place reading the [initMessages](https://github.com/search?q=repo%3Aapache%2Fsuperset%20initMessages&type=code) property so I removed it.